agent/bench-scripts/pbench-fio.md

```diff
@@ -200,21 +200,21 @@ Then you construct a fio job file for your initial sequential tests, and this wi
 
     [global]
     # size of each FIO file (in MB)
-    size=$@
+    size=1GiB
```

Comment:
This seems incorrect, given the comment on line 199.

Reply:
The size is not filled as well.